awswrangler/distributed/ray/modin/s3/_read_parquet.py

```diff
@@ -1,17 +1,48 @@
 """Modin on Ray S3 read parquet module (PRIVATE)."""
+import logging
 from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Union
 
 import modin.pandas as pd
 import pyarrow as pa
+import pyarrow.parquet
 from ray.data import read_datasource
 
+from awswrangler.distributed.ray import ray_remote
 from awswrangler.distributed.ray.datasources import ArrowParquetDatasource
 from awswrangler.distributed.ray.modin._utils import _to_modin
+from awswrangler.s3._read_parquet import _pyarrow_parquet_file_wrapper
 
 if TYPE_CHECKING:
     from mypy_boto3_s3 import S3Client
 
 
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+@ray_remote()
+def _read_parquet_metadata_file_distributed(
+    s3_client: Optional["S3Client"],
+    path: str,
+    s3_additional_kwargs: Optional[Dict[str, str]],
+    use_threads: Union[bool, int],
+    version_id: Optional[str] = None,
+    coerce_int96_timestamp_unit: Optional[str] = None,
+) -> Optional[pa.schema]:
+    fs = pyarrow.fs.S3FileSystem()
```

Comment:
My [PR](https://github.com/aws/aws-sdk-pandas/blob/feat/ray-s3fs/awswrangler/distributed/ray/s3/_list.py#L29) uses a PyArrow method to do this more cleanly

Reply:
Nice, thanks!